modules/sql/src/main/resources/db/migration/V058__Dhs_KeywordNames.sql

```diff
@@ -0,0 +1,18 @@
+---------------------------
+-- DHS Keyword Names
+-- This table list the DHS exceptions, of keywords names that don't follow
+-- the keyword name restrictions
+---------------------------
+
+CREATE TABLE e_dhs_keyword_names (
+    keyword identifier  PRIMARY KEY NOT NULL REFERENCES e_fits_keyword_names ON DELETE CASCADE,
+    name text             NOT NULL CHECK (name ~ '[a-zA-Z0-9_-]*')
```

Comment:
So the reason this is `text` is that ait can be longer than 8 chars, but does it need to be `text`. Could it be `VARCHAR(20)` or the max allowed by the DHS?

Reply:
You'll make me look into the DHS code to search for the limit

Reply:
How disappointing, I thought you'd know that kind of things by heart